plugins/timertop/timertopwidget.cpp

```diff
@@ -40,6 +40,74 @@
 
 using namespace GammaRay;
 
+class ClientTimerModel : public QSortFilterProxyModel
+{
+public:
+    explicit ClientTimerModel(QObject *parent = nullptr)
+        : QSortFilterProxyModel(parent)
+    { }
+
+    QVariant data(const QModelIndex &index, int role = Qt::DisplayRole) const override
+    {
+        if (hasIndex(index.row(), index.column()) && role == Qt::DisplayRole) {
+            switch (static_cast<TimerModel::Columns>(index.column())) {
+            case TimerModel::ColumnCount:
+                Q_ASSERT(false);
+            case TimerModel::ObjectNameColumn:
+            case TimerModel::TimerIdColumn:
+            case TimerModel::TotalWakeupsColumn:
+                // Use source model data
+                break;
+            case TimerModel::StateColumn:
+                return stateToString(QSortFilterProxyModel::data(index, role).toUInt());
+            case TimerModel::WakeupsPerSecColumn:
+                return wakeupsPerSecToString(QSortFilterProxyModel::data(index, role).toReal());
+            case TimerModel::TimePerWakeupColumn:
+                return timePerWakeupToString(QSortFilterProxyModel::data(index, role).toReal());
+            case TimerModel::MaxTimePerWakeupColumn:
+                return maxWakeupTimeToString(QSortFilterProxyModel::data(index, role).toUInt());
+            }
+        }
+
+        return QSortFilterProxyModel::data(index, role);
+    }
+
+    static QString stateToString(uint state)
+    {
+        const int timerState = (state & 0xffff);
+        const int timerInterval = (state >> 16);
+
+        switch (timerState) {
+        case 0: // None
+            return QCoreApplication::translate("TimerModel", "None");
```

Comment:
This should be tr(), avoids manually passing the context (which would need to include the namespace prefix btw), but more importantly, only tr() is extracted for translation, and I think you removed the tr() calls in the model previously providing the strings.